src/components/App/App.js

```diff
@@ -7,9 +7,8 @@ import { benefits } from '../../utils/config';
 function App() {
   return (
     <>
-      <Button label={'Sample Button'}/>
-      <Section type='benefits' layout='four-columns' title='Benefits'>
-        <CardList cards={benefits}></CardList>
+      <Section type='benefits' layout='horizontal' title='Benefits'>
+        <CardList cards={benefits} type='benefits'></CardList>
```

Comment:
Could we close these out just as `<CardList />` instead of having individual opening and closing component tags? Feel free to fact check that, I could be wrong!

Reply:
Fixed